tools/debian/rules

```diff
@@ -12,6 +12,10 @@ endif
 %:
 	dh $@
```

Comment:
Instead of the clean hack below, try to add `--buildsystem=autoconf` here. That should skip the auto-detection and preferring setup.py.

Reply:
YES!  This is what I was looking for, thanks.

Reply:
It worked.